src/ui/LiveSplit.tsx

```diff
@@ -295,10 +288,15 @@ export class LiveSplit extends React.Component<{}, State> {
         );
     }
 
-    public openTimerView() {
+    public openTimerView(remount: boolean) {
         this.setState({
             menu: { kind: MenuKind.Timer },
+            sidebarOpen: false,
         });
+        if (remount) {
+            this.state.layout.remount();
+            this.state.hotkeySystem.activate();
+        }
```

Comment:
Do we know why the others don't need it? This seems like a hacky workaround this way. They all need to remount the layout. Or at least they should. I don't see why the Run Editor and the About view would behave differently here. Neither is showing the layout.

Reply:
Oh wait, this is just so the submenus don't remount right? I guess we do the run editor and layout editor remounting somewhere else then?!

Reply:
The run editor and layout editor have their own methods (`closeRunEditor` and `closeLayoutEditor`). `openTimerView` is only used when you press the "Back" button in the sidebar in the Splits, Layout, and About views.

Reply:
It's still questionable that out of all three only the about view is doing the remounting this way. I feel like we should unify them better.